core/tests/protractor_utils/ExplorationEditorHistoryTab.js

```diff
@@ -202,10 +202,10 @@ var ExplorationEditorHistoryTab = function() {
       expectTextToMatch: function(v1StateContents, v2StateContents) {
         forms.CodeMirrorChecker(
           element.all(by.css('.CodeMirror-code')).first()
-        ).expectTextToBe(v1StateContents);
+        ).expectTextToBe(v1StateContents, 'first()');
```

Comment:
Why are we passing an extra parameter to `expectTextToBe`?
I'm not sure about the reason can you explain? Also, if required then you can rename it to `expecteFirstTextToBe` ditto fo below.

Reply:
Hi, @DubeySandeep 
the second parameter in `expectTextToBe` tells us to which Pane we have to match the test.
we have two panes first and last
Also, if we rename I believe the function we need to make two different functions which do the same?

Reply:
Can we initialize `CodeMirrorChecker` with two parameters? (element, pane)

Reply:
That's a good idea I've implemented the same :)